sdk/python/tests/integration/e2e/test_python_feature_server.py

```diff
@@ -25,64 +24,62 @@
 
 @pytest.mark.integration
 @pytest.mark.universal
-def test_get_online_features():
-    with setup_python_fs_client() as client:
-        request_data_dict = {
-            "features": [
-                "driver_stats:conv_rate",
-                "driver_stats:acc_rate",
-                "driver_stats:avg_daily_trips",
-            ],
-            "entities": {"driver_id": [5001, 5002]},
-        }
-        response = client.post(
-            "/get-online-features", data=json.dumps(request_data_dict)
-        )
+def test_get_online_features(python_fs_client):
```

Comment:
I don't see `python_fs_client` being defined anywhere?

Reply:
it's defined below in the same file